src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -341,7 +308,7 @@ private synchronized boolean isTunnelActive(final String tunnelId) {
 
   private boolean isServerReachable(final String host, final int port) {
```

Comment:
I don't think this is needed, and it forces the app to dig into the config. We should get rid of it.